.cbmc-batch/jobs/aws_cryptosdk_sig_get_privkey/aws_cryptosdk_sig_get_privkey_harness.c

```diff
@@ -23,28 +23,26 @@
 #include <cipher_openssl.h>
 
 void aws_cryptosdk_sig_get_privkey_harness() {
-    /* arguments */
-    struct aws_cryptosdk_sig_ctx *ctx = can_fail_malloc(sizeof(struct aws_cryptosdk_sig_ctx));
+    /* Nondet Input */
+    struct aws_cryptosdk_sig_ctx *ctx = ensure_nondet_sig_ctx_has_allocated_members();
     struct aws_allocator *alloc       = can_fail_allocator();
     struct aws_string *priv_key;
```

Comment:
You need to properly allocate this string here.
```c
struct aws_string *priv_key = malloc(sizeof(*priv_key));
```
And use the pointer when calling `aws_cryptosdk_sig_get_privkey`; otherwise, you missed the case where it's null.